cassandra/src/main/scala/geotrellis/spark/io/cassandra/CassandraLayerHeader.scala

```diff
@@ -0,0 +1,36 @@
+package geotrellis.spark.io.cassandra
+
+import geotrellis.spark.io.LayerHeader
+
+import spray.json._
+
+case class CassandraLayerHeader(
+  keyClass: String,
+  valueClass: String,
+  tileTable: String
```

Comment:
I think we need to store keyspace here as well. The idea behind the header is that it has enough information to locate the records by itself. AFAIK tables are not unique among keyspaces. Is that right?